### PR TITLE
WIP: Habla todo 

### PR DESCRIPTION
**Pull request by slytter**
A sub-master to merge all issues relating #9 (todo)

### CODE REVIEW

File: lib/todo/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const axios = require('axios');
+const _ = require('lodash');
+const NodeTrello = require('node-trello');
+
+const gitlab = axios.create({
+	baseURL: 'https://git.bitkompagniet.dk/api/v3/',
+	timeout: 10000,
+	headers: {
+		"PRIVATE-TOKEN": "-ypSz2utssd2zyhJsKmT", //eslint-ignore-quotes
```
Comment (kparkov) [inline lib/todo/index.js]: PAS nu på. Det er et offentligt repo, @neheren, og et private token.

File: lib/todo/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	},
+});
+
+const github = axios.create({
+	baseURL: 'https://api.github.com/',
+	timeout: 1000,
+	headers: {
+		Authorization: 'Basic bmlrb2xhai5zbkBob3RtYWlsLmNvbTpLb21ta29HSDE',
```
Comment (kparkov) [inline lib/todo/index.js]: Igen...

File: lib/todo/index.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	},
+});
+
+// nodeTrello is used as a wrapper for the trello api library.
+const token = 'f1a0198c81c817bcad472ced8b2a09132fbe8238656fdd64081f475f3b8dc6bf';
```
Comment (kparkov) [inline lib/todo/index.js]: Igen igen...

File: lib/todo/index.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+const key = '162086b7db000391513e98bcbc35cda7';
```
Comment (kparkov) [inline lib/todo/index.js]: Igen igen igen...

**Comment by slytter** [comment]
@kparkov Habla todo is basically done. Should I merge?

**Comment by kparkov** [comment]
If by "basically", you mean "actually", I think you should merge. :-) I am not going to review the code, it's too massive. Instead, I will test and submit issues.

**Pull request merged**